### ISSUE TITLE
[css-viewport] Use ASCII whitespace

### ISSUE THREAD

**Issue by annevk** [w3c/csswg-drafts#8757]
Since this is an HTML feature it would make sense to use https://infra.spec.whatwg.org/#ascii-whitespace for the whitespace definition.

**Comment by annevk** [comment]
I would love to get a resolution on this somehow, asynchronous is fine.

**Comment by astearns** [comment]
So the effect of this change would be to add `U+000C FF` to the whitespace set. Maybe this could be done async?

**Comment by annevk** [comment]
Yeah that's correct. It should probably just reference the Infra ASCII whitespace definition.

**Comment by astearns** [comment]
The CSSWG will automatically accept this resolution one week from now if no objections are raised here. Anyone can add an emoji to this comment to express support. If you do not support this resolution, please add a new comment.

Proposed Resolution: Define whitespace for `<meta>` parsing using WHATWG infra (adding `U+000C FF`)

**Comment by Loirooriol** [comment]
Both the proposed resolution and this issue in general don't seem very clear to me.
What is this about, parsing the `content` attribute of a `<meta name=viewport>`?

**Comment by astearns** [comment]
Yes. My understanding is that we would replace the whitespace list in https://drafts.csswg.org/css-viewport-1/#parsing-algorithm with a link to https://infra.spec.whatwg.org/#ascii-whitespace, which has the effect of adding `U+000C FF` to the list

**Comment by astearns** [comment]
RESOLVED: Define whitespace for `<meta>` parsing using WHATWG infra (adding `U+000C FF`)

**Comment by annevk** [comment]
If anyone has a good way of testing this I'd be happy to add a test, but it seems we don't have any parsing tests currently and WebKit's tests rely on internal APIs. (It seems that the existing WPT `<meta name=viewport>` tests might just pass everywhere because they don't actually test anything.)
